scrapscript.py

```diff
@@ -374,26 +367,36 @@ def parse(tokens: typing.List[Token], p: float = 0) -> "Object":
         tokens.pop(0)
     elif isinstance(token, LeftBracket):
         l = List([])
-        token = tokens[0]
-        if isinstance(token, RightBracket):
-            tokens.pop(0)
-        else:
-            l.items.append(parse(tokens, 2))
-            while not isinstance(tokens.pop(0), RightBracket):
-                # TODO: Implement .. and ... operators
+        while True:
+            # TODO: Implement .. operator
+            if isinstance(tokens[0], RightBracket):
```

Comment:
I think you should keep the older structure (see my branch). As it is, your code seems to allow `[,,,,,,]`